### ISSUE TITLE
Error when running poller in production 

### ISSUE THREAD

**Issue by douglascamata** [telegram-bot-rb/telegram-bot#3]
```
RAILS_ENV=production bundle exec rake telegram:bot:poller
rake aborted!
Poller not found for :default
/usr/local/opt/rbenv/versions/2.3.0/gemsets/mastermind/gems/telegram-bot-0.6.0/lib/telegram/bot/updates_poller.rb:21:in `start'
/usr/local/opt/rbenv/versions/2.3.0/gemsets/mastermind/gems/telegram-bot-0.6.0/lib/tasks/telegram-bot.rake:7:in `block (3 levels) in <top (required)>'
/usr/local/opt/rbenv/versions/2.3.0/bin/bundle:23:in `load'
/usr/local/opt/rbenv/versions/2.3.0/bin/bundle:23:in `<main>'
Tasks: TOP => telegram:bot:poller
(See full trace by running task with --trace)
```

When running in the development environment everything works fine.

Secrets are as follows:

```
development:
  secret_key_base: d873a75e86e73cb5f0f18961b08d2ea721fe676b050e35caeee3c1e8afb96f62a76f7c8f041417f4f26dc1895831c8dc492589a5f599d14155ae8cf8a33b2e2b
  telegram:
    bot:
      token: <%= ENV["TELEGRAM_BOT_TOKEN"] %>
      username: bot_name

test:
  telegram:
    bot:
      token: <%= ENV["TELEGRAM_BOT_TOKEN"] %>
      username: bot_name
  secret_key_base: cd7f42ca1d03372280e76cd0e073970bd3522f5986f6a2eccba26782ac66ffcd30655abea290aba9291ec8b75b690b1d3ccfe1451942367453e9d4e867e58466

production:
  telegram:
    bot:
      token: <%= ENV["TELEGRAM_BOT_TOKEN"] %>
      username: bot_name
  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
```

**Comment by printercu** [comment]
oh, I haven't tested poller mode in production. Thought nobody will use it :)
You can workaround it by adding `Telegram.bot_poller_mode = true` to `application.rb` or initializer.

I'll try to find out how to make it work with `spring` or will just add notice to readme.

**Comment by douglascamata** [comment]
@printercu I'm using it for a demo of a project that I built in a hackathon. My deploy setup is not 100% adequate for real production environments because I have no domain and it's an experimental setup, so IPs and ports might change. So I thought it would be better to use the poller in this specific situation. 

Anyway, I figured the `Telegram.bot_poller_mode` thing after some investigation. I kept the issue here because it should be documented, IMHO. I also noticed that I get double logging for everything when `Telegram.bot_poller_mode` is set and `RAILS_ENV=production`. Looks like everything is being logged to stdout AND stderr.

**Comment by printercu** [comment]
Yeah, I've set it without respect for `Rails.env`: https://github.com/telegram-bot-rb/telegram-bot/blob/master/lib/tasks/telegram-bot.rake#L6 :( Maybe i'll use envvar to configure it. 

For now the best way is to define your own task:
```ruby
    task my_poller: :environment do
      Telegram::Bot::UpdatesPoller.start(ENV['BOT'].try!(:to_sym) || :default)
    end
```

**Comment by douglascamata** [comment]
I see. That can be easily solved: 

`Rails.logger.extend ActiveSupport::Logger.broadcast console unless Rails.env.production?`

**Comment by printercu** [comment]
Yep. But I think about cases when someone will want to see log in console in production or staging env.

**Comment by douglascamata** [comment]
IMHO, you should keep the telegram bot logger separated from Rails logging. Requests that come into the webhook controller would be logged by Rails and when in poller mode Rails logging isn't used at all.

**Comment by printercu** [comment]
What do you mean "polling log"? Poller logs only on start and stop.

**Comment by douglascamata** [comment]
@printercu by "polling log" I mean all the logs coming from messages from Telegram that are processed while the bot is in polling mode.

**Comment by printercu** [comment]
The are logged by controller. Like `params` in usual controller.

**Comment by printercu** [comment]
@douglascamata fixed it in 0.7.3. Everything should work just as expected.

**Comment by douglascamata** [comment]
@printercu thank you very much!
